### PR TITLE
Add woox apis

### PR DESCRIPTION
**Pull request by sc0Vu**
WOOX had added multiple api endpoints for funding rate and futures: https://kronosresearch.github.io/wootrade-documents, in this PR, I'll implement this api and add functions related to the api, eg fetchFundingRate.

These functions are supposed to be related to swap / future / perp, need to check whether there is api, TODO List:

- [x] fetchFundingHistory
- [x] fetchFundingRate
- [x] fetchFundingRateHistory
- [x] fetchFundingRates
- [x] fetchLeverage
- [x] setLeverage
- [x] fetchPosition
- [x] fetchPositions
- [x] fetchMarkets
- [x] createOrder
- [x] fetchOrder
- [x] fetchOrders
- [x] cancelOrder
- [x] cancelOrders
- [ ] ~fetchLeverageTiers~
- [ ] ~setMargin~
- [ ] ~setPositionMode~
- [ ] ~setMarginMode~
- [ ] ~fetchMarketLeverageTiers~
- [ ] ~createReduceOnlyOrder~
- [ ] ~fetchPositionsRisk~
- [ ] ~fetchMarkOHLCV~
- [ ] ~fetchIndexOHLCV~
- [ ] ~fetchPremiumIndexOHLCV~
- [ ] ~reduceMargin~
- [ ] ~addMargin~
- [ ] ~modifyMarginHelper~

### CODE REVIEW

**Comment by frosty00** [comment]
is it possible to add the unified methods to this PR also?

**Comment by sc0Vu** [comment]
I'm working on it!

File: js/woo.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = class woo extends Exchange {
                 'spot': true,
                 'margin': true,
                 'swap': false,
-                'future': false,
+                'future': true,
```
Comment (Dan-krm) [inline js/woo.js:27]:
```
‘future’: false,
‘swap’: true,
```
All of the futures trading pairs are perpetual futures (what we call swap) future should only be set to true if there are traditional delivery futures with an expiration date available. Set future to false and set swap to true.

File: js/woo.js [hunk 2]
```diff
@@ -93,14 +97,15 @@ module.exports = class woo extends Exchange {
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/150730761-1a00e5e0-d28c-480f-9e65-089ce3e6ef3b.jpg',
                 'api': {
+                    'pub': 'https://api-pub.staging.woo.org',
```
Comment (kroitor) [inline js/woo.js:100]: We cannot use the staging API URL here, it should be a production API URL instead.

**Pull request merged**